en/siddhi-examples/examples/value-based-filtering/value_based_filtering.out

```diff
@@ -0,0 +1,7 @@
+#### Events 
+
+Events  will arrive at the following streams only if the input of `TemperatureStream` equals,
```

Comment:
Talk about the input event and then explain the output

Reply:
Fixed in 72084b8